### PR TITLE
chore: change askx to ask

### PR DESCRIPTION
**Pull request by pbheemag**
*Issue #, if available:*

*Description of changes:*
Change askx to ask for all commands.
Update `v2new` to just `new`
Update github url (from alexa-labs to alexa) in links across the repo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by pbheemag** [comment]
> the “ask-cli-x” in the documents are needed to change to "ask-cli"

addressed.

File: lib/controllers/skill-infrastructure-controller/deploy-delegate.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const BUILT_IN_DEPLOY_DELEGATES = {
         location: 'cfn-deployer',
         description: CONSTANTS.DEPLOYER_TYPE.CFN.DESCRIPTION,
         templates: {
-            branch: 'ask-cli-x',
+            branch: 'ask-cli',
```
Comment (ChihYing7509) [inline lib/controllers/skill-infrastructure-controller/deploy-delegate.js:10]: The branch name hasn't been updated, so it should be 'ask-cli-x' for now

File: lib/utils/constants.js [hunk 2]
```diff
@@ -184,7 +184,7 @@ module.exports.FILE_PATH = {
 };
 
 module.exports.TEMPLATES = {
-    TEMPLATE_BRANCH_NAME: 'ask-cli-x',
+    TEMPLATE_BRANCH_NAME: 'ask-cli',
```
Comment (ChihYing7509) [inline lib/utils/constants.js:187]: The branch name hasn't been updated, so it should be 'ask-cli-x' for now

File: lib/controllers/skill-infrastructure-controller/deploy-delegate.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ const BUILT_IN_DEPLOY_DELEGATES = {
         location: 'lambda-deployer',
         description: CONSTANTS.DEPLOYER_TYPE.LAMBDA.DESCRIPTION,
         templates: {
-            branch: 'ask-cli-x',
+            branch: 'ask-cli',
```
Comment (ChihYing7509) [inline lib/controllers/skill-infrastructure-controller/deploy-delegate.js:22]: the same with the above

File: test/unit/commands/v2new/helper-test.js [hunk 4]
```diff
@@ -3,13 +3,13 @@ const fs = require('fs-extra');
...
-const Manifest = require('@src/model/manifest');
-const stringUtils = require('@src/utils/string-utils');
-const CONSTANTS = require('@src/utils/constants');
+const GitClient = require('./node_modules/@src/clients/git-client');
+const SkillInfrastructureController = require('./node_modules/@src/controllers/skill-infrastructure-controller');
+const helper = require('./node_modules/@src/commands/new/helper');
+const ResourcesConfig = require('./node_modules/@src/model/resources-config');
+const Manifest = require('./node_modules/@src/model/manifest');
+const stringUtils = require('./node_modules/@src/utils/string-utils');
+const CONSTANTS = require('./node_modules/@src/utils/constants');
```
Comment (kakhaUrigashvili) [inline test/unit/commands/v2new/helper-test.js:12]: ./node_modules/ part looks wrong

File: README.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <br/>
   <h1 align="center">Alexa Skills Kit Command Line interface</h1>
   <p align="center">
-    <a href="https://www.npmjs.com/package/ask-cli-x"><img src="https://badge.fury.io/js/ask-cli-x.svg"></a>
+    <a href="https://www.npmjs.com/package/ask-cli"><img src="https://badge.fury.io/js/ask-cli-x.svg"></a>
     <a href="https://travis-ci.org/alexa-labs/ask-cli"><img src="https://travis-ci.org/alexa-labs/ask-cli.svg?branch=master"></a>
```
Comment (RonWang) [inline README.md:7]: alexa-labs need to change to alexa

File: package.json [hunk 6]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "ask-cli-x",
+  "name": "ask-cli",
   "version": "0.12.0",
   "description": "Alexa Skills Kit (ASK) Command Line Interfaces - the X version",
```
Comment (RonWang) [inline package.json:4]: version need a bump; remove the X version in the description

File: docs/Upgrade-Project-From-V1.md [hunk 7]
```diff
@@ -1,20 +1,20 @@
-# Upgrade Guide For ask-cli beta (askx)
- 
-> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI Beta (askx)**
- 
-Skills created with CLI 1.x will need to update their project structure in order to deploy with the CLI Beta (askx). To help projects upgrade to the new format, we have provided an `askx util upgrade-project` command that will attempt to perform the necessary changes.  
- 
+# Upgrade Guide For ask-cli beta (ask)
```
Comment (RonWang) [inline docs/Upgrade-Project-From-V1.md:1]: I think those are not beta anymore. All the beta word in this page need to be removed

File: docs/Upgrade-Project-From-V1.md [hunk 8]
```diff
@@ -1,20 +1,20 @@
-# Upgrade Guide For ask-cli beta (askx)
- 
-> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI Beta (askx)**
- 
-Skills created with CLI 1.x will need to update their project structure in order to deploy with the CLI Beta (askx). To help projects upgrade to the new format, we have provided an `askx util upgrade-project` command that will attempt to perform the necessary changes.  
- 
+# Upgrade Guide For ask-cli (ask)
+
+> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI (ask)**
```
Comment (RonWang) [inline docs/Upgrade-Project-From-V1.md:3]: I think the previous beta should be replaced by "v2" as this doc is to upgrade project from v1 to v2. This place (ask) should actually be `with the CLI v2`

File: docs/Upgrade-Project-From-V1.md [hunk 9]
```diff
@@ -1,20 +1,20 @@
...
+
+Skills created with CLI 1.x will need to update their project structure in order to deploy with the CLI (ask). To help projects upgrade to the new format, we have provided an `ask util upgrade-project` command that will attempt to perform the necessary changes.
```
Comment (RonWang) [inline docs/Upgrade-Project-From-V1.md:5]: Same, should be `with the CLI v2`

File: docs/Upgrade-Project-From-V1.md [hunk 10]
```diff
@@ -1,20 +1,20 @@
...
-    `$ askx util upgrade-project` 
+2. Install ask-cli:
+    `$ npm install -g ask-cli`
+3. Upgrade your skill project with ask-cli. From your project's root, run:
+    `$ ask util upgrade-project`
     * The command will make the following changes to the project structure:
-    
-        |                      | v1 project        | beta (askx) project                        |
+
+        |                      | v1 project        | (ask) project                        |
```
Comment (RonWang) [inline docs/Upgrade-Project-From-V1.md:17]: Similar, this is also a `v2 project` as table header

File: package.json [hunk 11]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "ask-cli-x",
+  "name": "ask-cli",
   "version": "0.13.0",
```
Comment (RonWang) [inline package.json:3]: we can change the version to 2.0.0 within this branch as of now

**Pull request merged**